src/test/java/org/openelisglobal/provider/ProviderServiceTest.java

```diff
@@ -0,0 +1,56 @@
+package org.openelisglobal.provider;
+
+import java.text.ParseException;
+import org.junit.After;
+import org.junit.Before;
+import org.openelisglobal.BaseWebContextSensitiveTest;
+import org.openelisglobal.person.service.PersonService;
+import org.openelisglobal.person.valueholder.Person;
+import org.openelisglobal.provider.service.ProviderService;
+import org.openelisglobal.provider.valueholder.Provider;
+import org.springframework.beans.factory.annotation.Autowired;
+
+public class ProviderServiceTest extends BaseWebContextSensitiveTest {
+
+    @Autowired
+    ProviderService providerService;
+
+    @Autowired
+    PersonService personService;
+
+    @Before
+    public void init() throws Exception {
+        providerService.deleteAll(providerService.getAll());
+    }
+
+    @After
+    public void tearDown() throws Exception {
+        providerService.deleteAll(providerService.getAll());
+    }
+
+    // @Test
+    // public void createProvider_shouldReturnCreatedProvider() throws Exception{
+    // String firstName = "John";
```

Comment:
did you intend to work on this but forgot?

Reply:
which is why the build is failing as can be seen at https://github.com/I-TECH-UW/OpenELIS-Global-2/actions/runs/11707145829/job/32605834580?pr=1302#step:8:10974

Reply:
Hello @mherman22 
i think can be resolved by updating this specific branch but that doesn't solve the problem i found out earlier on

Reply:
Can you click this link and see what its saying https://github.com/I-TECH-UW/OpenELIS-Global-2/actions/runs/11834718253/job/32975979073?pr=1302#step:8:11003

Reply:
yeah your right let me look into it shortly

Reply:
Is the PR title right? adding an integration test for provider server? or provider test?

Reply:
thank you for that correction @mherman22